plugins/protoc-gen-json.py

```diff
@@ -16,25 +16,28 @@ def log(*args):
     print(*args, file=sys.stderr)
 
 
-def traverse(proto_file):
```

Comment:
Removed iterator and simplified to a returned list. Probably uses more memory but is easier to reason about.